src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -425,6 +425,7 @@ static std::string parseMqttQoS(ConnectionInfo* ciP, SubscriptionUpdate* subsP,
 */
 static std::string parseMqttTopic(ConnectionInfo* ciP, SubscriptionUpdate* subsP, const Value& mqtt)
 {
+  // FIXME PR: check topic doesn't contain # or + (add new steps to ftest)
```

Comment:
This is not going to be solved in this PR (it is out of scope) but I marked it as FIXME PR so I don't forget in the main PR #3888 

NTC (informative)